src/Swift.Bindings/tests/IntegrationTests/Swift.Bindings.Integration.Tests.csproj

```diff
@@ -21,6 +22,17 @@
     <!-- Generate Swift libraries and projection tooling bindings -->
     <Exec Command="$(PreTestCommand)" />
```

Comment:
Alternatively, you can use `ConsoleToMsBuild` which captures the output as an item: https://learn.microsoft.com/en-us/visualstudio/msbuild/exec-task?view=vs-2022
```
<Exec Command="$(PreTestCommand)"
      ConsoleToMSBuild="true">
  <Output TaskParameter="ConsoleOutput" ItemName="CMakeOutputLines"/>
</Exec>
```

Reply:
Actually having the output in a log file helps, since the cmake warnings are multi-line and thus not easily "parsable" via regexes and such. So the detection I added is to just notice a warning - then go to the log and figure out what the problem realy is.
The full log does show up in binlog as well if we need to get it from there.